logistics/scripts/modules/settings/settings-management.psm1

```diff
@@ -312,21 +321,75 @@ function New-DevelopmentAppSettings([hashtable] $Settings = @{ }) {
         $developmentSettingsForEngine = (Get-DefaultDevelopmentSettingsByEngine)[$Settings.ApiSettings.Engine]
         $developmentSettingsForEngine = Add-ApplicationNameToConnectionStrings $developmentSettingsForEngine $project
 
-        $newDevelopmentSettings = Merge-Hashtables $developmentSettingsByProject[$project], $developmentSettingsForEngine, $Settings
+        $newDevelopmentSettings = Merge-Hashtables $developmentSettingsByProject[$project], $developmentSettingsForEngine
 
         $newDevelopmentSettings = Add-TestHarnessSpecificAppSettings $newDevelopmentSettings $project
 
+        $newDevelopmentSettings = Merge-Hashtables $newDevelopmentSettings, (Get-CredentialSettingsByProject)[$project], $Settings
+
         $projectPath = Get-RepositoryResolvedPath $Project
 
         $newDevelopmentSettingsPath = Join-Path $projectPath "appsettings.development.json"
         New-JsonFile $newDevelopmentSettingsPath $newDevelopmentSettings -Overwrite
         $newSettingsFiles += $newDevelopmentSettingsPath
 
         $credentialSettings = Merge-Hashtables @{ }, (Get-CredentialSettingsByProject)[$project]
-        $newUserSettingsPath = Join-Path $projectPath "appsettings.user.json"
-        New-JsonFile $newUserSettingsPath $credentialSettings
-        $newSettingsFiles += $newUserSettingsPath
+        $userSecrets += ($credentialSettings | ConvertTo-Json -Depth 10 | dotnet user-secrets set --project $projectPath --id (Get-UserSecretsIdByProject).$project)
```

Comment:
As discussed we should put our default credential settings in the development json file and leave the user secrets file for any user overrides to those settings. The main reason being that we don't want to be replacing the user secrets with our settings every time we run initdev.

Reply:
I understand it now . I remove that code .Thanks